dotCMS/src/main/java/com/dotmarketing/portlets/workflows/util/WorkflowEmailUtil.java

```diff
@@ -61,7 +55,7 @@ public class WorkflowEmailUtil {
 	 */
 
     public static void sendWorkflowEmail(WorkflowProcessor processor, String[] email, String subject, String emailText,
```

Comment:
As it is a public method, I would overload the method instead of overriding it. We don't know if a customer is using this method, and we don't want to break existing implementations